README.md

```diff
@@ -25,8 +25,8 @@ class ViewController: FormViewController {
             	row.displayValueFor = { "\($0 ?? 0)" }
             }
             <<< TableInlineRow<String> { row in
-                row.options = ["first", second", "third"]
-				row.value = "none"
+                row.options = ["first", "second", "third"]
+		row.value = "none"
```

Comment:
I think you should use spaces and not tabs here, same as in the line above

Reply:
good call